bin/node/src/chain_spec.rs

```diff
@@ -129,37 +118,49 @@ pub struct ChainParams {
     #[structopt(long, default_value = "account_secret")]
     pub account_key_file: String,
 
+    // The length of a session
     #[structopt(long)]
     pub session_period: Option<u32>,
 
+    /// Time interval (in milliseconds) between blocks
     #[structopt(long)]
     pub millisecs_per_block: Option<u64>,
 
+    /// Chain name
     #[structopt(long, default_value = "Aleph Zero Development")]
     pub chain_name: String,
 
+    /// Token symbol
     #[structopt(long, default_value = "DZERO")]
     pub token_symbol: String,
 
-    /// Pass the AccountIds of authorities forming the committe at the genesis
-    ///
-    /// Expects a delimited collection of AccountIds
-    #[structopt(long, require_delimiter = true, parse(from_str = parse_account_id), required_unless("dev-chain"))]
+    /// AccountIds of authorities forming the committe at the genesis (comma delimited)
+    #[structopt(long, require_delimiter = true, parse(from_str = parse_account_id))]
     account_ids: Vec<AccountId>,
 
-    /// Pass the AccountId of the sudo account
-    ///
-    /// If the chain-id is "dev" it will default to the first generated account (Alice)
-    /// and use a default pre-defined id in any other case
+    /// AccountId of the sudo account
     #[structopt(long, parse(from_str = parse_account_id), default_value(DEFAULT_SUDO_ACCOUNT))]
     sudo_account_id: AccountId,
+
+    /// AccountId of the optional faucet account
+    #[structopt(long, parse(from_str = parse_account_id))]
+    faucet_account_id: Option<AccountId>,
 }
 
 impl ChainParams {
     pub fn chain_id(&self) -> &str {
         &self.chain_id
     }
 
+    pub fn chain_type(&self) -> ChainType {
+        match self.chain_id.as_str() {
```

Comment:
!critical typo! `check_type` - this is the reason why this still doesn't work for me :stuck_out_tongue_closed_eyes: